src/ray/common/ray_config_def.h

```diff
@@ -60,6 +60,11 @@ RAY_CONFIG(uint64_t, num_heartbeats_warning, 5)
 
 /// The duration between reporting resources sent by the raylets.
 RAY_CONFIG(uint64_t, raylet_report_resources_period_milliseconds, 100)
+RAY_CONFIG(uint64_t, raylet_report_loads_period_milliseconds, 1000)
+
+/// The duration between raylet check memory pressure and send gc request
+RAY_CONFIG(uint64_t, raylet_check_gc_period_milliseconds, 100)
```

Comment:
Was this hardcoded before?

Reply:
This was the same as `raylet_report_resources_period_milliseconds` before.